python/cudf/cudf/io/json.py

```diff
@@ -7,6 +7,7 @@
 import cudf
 from cudf._lib import json as libjson
 from cudf.utils import ioutils
+from cudf.utils.dtypes import is_list_like
 
 
 @ioutils.doc_read_json()
```

Comment:
Should we also add to the documentation?

Reply:
For sure. I'll add it as part of my next commit.